lib/paper_trail/version_concern.rb

```diff
@@ -376,7 +376,7 @@ def sibling_versions
     #
     # @api private
     def version_limit
-      klass = item.class
+      klass = item ? item.class : item_type.constantize
```

Comment:
here is the problem, `item` is nil when the class has `id_key`. I can't find another solution, so, suggestions are welcome